cmd/controller/app/options/options.go

```diff
@@ -222,6 +225,8 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringSliceVar(&s.EnabledControllers, "controllers", defaultEnabledControllers, ""+
 		"The set of controllers to enable.")
+	fs.StringSliceVar(&s.DisabledControllers, "disabled-controllers", defaultDisabledControllers, ""+
+		"The set of controllers to disable. Supersedes --controllers.")
```

Comment:
nit: I'm not sure if "supersedes" is the right word here:

> supersede: take the place of (a person or thing previously in authority or use); supplant.

I first read the comment as meaning:

> "The set of controllers to disable. Replaces --controllers."

But it doesn't really replace --controllers, since `EnabledControllers` is still used. Maybe:

> "The set of controllers to disable. Takes priority over --controllers."